monitor/monitoring-dam-generator-core/src/main/java/eu/seaclouds/monitor/monitoringdamgenerator/dcgenerators/MODACloudsDcGenerator.java

```diff
@@ -17,10 +15,10 @@
             .getLogger(MODACloudsDcGenerator.class);
 
     private static final String MODACLOUDS_DC_ID = "modacloudsDc";
-    private static final String START_SCRIPT_URL = "http://start.sh/";
+    private static final String START_SCRIPT_URL = "http://installMODACloudsDc.sh/";
```

Comment:
Yes, we need to publish these files. For example, we could create a new repository in SeaClouds organization which contains the required files, upload these files to SeaCloudsPlatform repository, Amazon S3...